src/Header/MobileMenu.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState } from "react";
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import listIcon from './list.svg';
+import listCloseIcon from './listClose.svg';
+
+interface MobileMenuProps {
+  links: string[];
+}
+
+interface MenuButtonProps {
+  getOpen: () => boolean;
+  handleToggle: () => void;
+}
+
+interface MenuListProps {
+  links: string[];
+  getOpen: () => boolean;
+  closeList: () => void;
+}
+
+interface StyledListProps {
+  isOpen: boolean;
+}
+
+const MobileMenu = ({ links }: MobileMenuProps) => {
+  const [open, setOpen] = useState(false);
+
+  const getOpen = () => open;
+  const handleToggle = () => {
+    console.log(open);
+    setOpen(!open);
+  }
+  return (
+    <div>
```

Comment:
Could we replace the div with a fragment?

Reply:
I tried it -- seems to break the styling.